### ISSUE TITLE
Artwork mixed up

### ISSUE THREAD

**Issue by ktkm** [Dicotype/Afacad#12]
@emmamarichal The images for Afacad Flux on GF mix new and old artwork. To better distinguish between the different families' versions of Afacad, I have decided to create a new set of artwork for each family.

**Comment by emmamarichal** [comment]
Alright, let me know once it's ready to use :) 
Please tell me also if there are different description for the about section!

**Comment by ktkm** [comment]
Yes, the second paragraph in the about section of Afacad Flux that starts with “Furthermore …” should read:
_‘Afacad Flux’ adds an extra dimension with a back-slanted version, commemorating, among other things, the typesetting of river names in historical cartography. Furthermore, the inclusion of supplementary weights and expanded language support contributes to a versatile typeface collection, well-suited for industrial and commercial applications._

**Comment by ktkm** [comment]
@emmamarichal New artwork is ready!

**Comment by emmamarichal** [comment]
Thank you, I'll do the update asap!

**Comment by ktkm** [comment]
Good morning @emmamarichal,

A couple of things:

1. With the risk of sounding tautological in the About section of Afacad Flux, I would like the first paragraph to be the same as in the Afacad About section -- by changing the phrase ’Afacad Flux typeface project’ to ’Afacad typeface project’.

_The ’Afacad typeface project’ commenced in 2017 as a personalised lettering endeavour for Slagskeppet, a Swedish housing tenant, who sought fresh house address numbering for their entrances. The letters and numerals were meticulously crafted to harmonise with the architectural proportions and materials employed by Architect Sture Elmén during the 1940s._

2. How long before the new artwork can be upgraded on each About section of Afacad and Afacad Flux? Also, when will the latest version of ‘normal’ Afacad be available on Google Fonts?

**Comment by ktkm** [comment]
@emmamarichal Did you see my last post?

**Comment by emmamarichal** [comment]
Yes, thank you, I'll do the changes this week or next week!

**Comment by emmamarichal** [comment]
Hi @ktkm,
I made the changes for articles, they should appear in a few weeks.
For the Afacad update, what are the changes from the previous version? I don't recall about an planned upgrade. Thanks!

**Comment by ktkm** [comment]
Hello @emmamarichal, Like Afacad Flux, Afacad has the same weight progression update in version 1.100—from Thin to Black. Remember? ;-)

**Comment by emmamarichal** [comment]
Ok thank you, I'll add it on the issue tracker, but since we have a lot of planned projects for Q4, it will probably go for Q1 2025. Thank you for your patience :)
